package-support-policy.md

```diff
@@ -0,0 +1,372 @@
+
+Additionally, you can find out which target frameworks are currently in-support by visiting the following pages:
+
+- .NET: https://dotnet.microsoft.com/en-us/platform/support/policy
+- .NET Framework: https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet-framework
+- .NET Standard: https://learn.microsoft.com/en-us/dotnet/standard/net-standard
+
+Keep in mind that the minimum .NET Standard version supported by a package depends on which minimum .NET and .NET Framework versions are supported by the package. Packages and projects targeting .NET Standard are expected to be compatible between each other by following the same rules specified by our .NET Standard policy.
+
+## Release cadence
+
+Some packages follow the same release cadence as the .NET release cadence, meaning we release a new version of the package each time we release a new .NET version. For example:
+
+- [System.Text.Json](https://www.nuget.org/packages/System.Text.Json), which is part of the shared framework, but is also available as a NuGet package.
+- [Microsoft.Extensions.Configuration.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Abstractions), which is exclusively released as a NuGet package.
+
+Note that the source code of those two packages is hosted in the [dotnet/runtime](https://github.com/dotnet/runtime) repo.
+
+On the other hand, the packages that come out from the maintenance-packages repo have a separate, independent release cadence, meaning their servicing releases will come out of this repo and will not be bound by any pre-defined release schedule.
+
+These are packages that originally had their source code hosted in branches that are already out of support, but the packages themselves are still in-support, because they target frameworks that are still supported. Some examples:
+
+- [System.Xml.XPath.XmlDocument](https://www.nuget.org/packages/System.Xml.XPath.XmlDocument) - Its source code used to be hosted in the .NET Core 1.1 branch.
+- [System.Buffers](https://www.nuget.org/packages/System.Buffers) - Its source code used to be hosted in the .NET Core 2.1 branch.
+- [Microsoft.Bcl.HashCode](https://www.nuget.org/packages/Microsoft.Bcl.HashCode) - Its source code used to be hosted in the.NET Core 3.1 branch.
+- [System.Net.WebSockets.WebSocketProtocol](https://www.nuget.org/packages/System.Net.WebSockets.WebSocketProtocol) - Its source code used to be hosted in the .NET 5.0 branch.
+
+## End of support
+
+A package will be supported for as long as it targets frameworks that are still supported. For example: [System.Numerics.Vectors](https://www.nuget.org/packages/System.Json#supportedframeworks-body-tab). Of all the included target frameworks in the package, .NET Standard 2.0 is still in support at the moment of writing this document. As long as there are frameworks compatible with .NET Standard 2.0, the package will remain in support.
```

Comment:
NETStandard is not sufficient here, since it's not a runnable framework.  We also don't explicitly "support" NETStandard with servicing because of this - you can't run code there. 

Packages are supported on the frameworks where they can actually run.  `System.Numerics.Vectors` is supported because it provides an implementation on `net462` which is still in support.

Reply:
> NETStandard with servicing because of this - you can't run code there.

Right, but that's why the last sentence says:

> As long as there are frameworks compatible with .NET Standard 2.0, the package will remain in support.

Would that be confusing to customers? What would you suggest instead?